environments/development/invoke.yml

```diff
@@ -5,6 +5,7 @@ django:
   debug: true
   postgres_user: "django"
   postgres_host: "surfpol-edushare.c85na2fjlzol.eu-central-1.rds.amazonaws.com"
+  redis_host: "redis.2w7uxm.ng.0001.euc1.cache.amazonaws.com:6379"
```

Comment:
Hoe heb je die opgetuigd? Via terraform?

Reply:
Nee, handmatig. Ticket voor Terraform staat open. Die wilde ik eerst oppakken, maar ga nu eerst urgente search bugs fixen, waardoor materialen niet gevonden worden. Daarna kan ik wel kijken hoe het in Terraform moet met ElastiCache.